provider/rancher.go

```diff
@@ -290,7 +290,10 @@ func listRancherEnvironments(client *rancher.RancherClient) []*rancher.Environme
 
 	var environmentList = []*rancher.Environment{}
 
-	environments, err := client.Environment.List(nil)
+	withoutPagination := &rancher.ListOpts{
```

Comment:
Sure @timoreimann. Like [this](https://github.com/containous/traefik/pull/1453/commits/bf331f71bcba22b44c6f70c20430ae560847f4bb)?